include/valkey/valkey.h

```diff
@@ -270,6 +270,12 @@ typedef struct valkeyContextFuncs {
      * these functions shall return a value < 0.  In the event of a
      * recoverable error, they should return 0. */
     ssize_t (*read)(struct valkeyContext *, char *, size_t);
+    /* ZC means zero copy, it provides underlay transport layer buffer directly,
+     * so it has better performance than generic read. After consuming the read
+     * buffer, it's necessary to notify the underlay transport to advance the
+     * read buffer by read_zc_done. */
+    ssize_t (*read_zc)(struct valkeyContext *, char **);
+    ssize_t (*read_zc_done)(struct valkeyContext *);
```

Comment:
Using an int here would better describe that only a status code is returned.
WDYT?
```suggestion
    int (*read_zc_done)(struct valkeyContext *);
```

Same below in `static ssize_t valkeyRdmaReadZCDone`

Reply:
I notice this function family also use ssize_t, so I think ssize_t for read_zc_done will be better.
```
    ssize_t (*read)(struct valkeyContext *, char *, size_t);
    ssize_t (*write)(struct valkeyContext *);
```